2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,28 @@
 function populateTodoList(todos) {
+  // Access to the list div
   let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  // Add class to list
+  list.classList = "list-sec";
+  // Remove all list before recreating
+  list.textContent = "";
+  // Loop throug to our todos
+  todos.forEach((todo) => {
+    // Create li tag
+    const li = document.createElement("li");
```

Comment:
Since li-sec and list-sec are the same things. To avoid confusion, it would be beneficial to use a more specific naming convention, such as "todo-list-sec," to differentiate it from other similarly named sections. This will be especially important for other engineers who will be working with same codebase for example, Comments should only be used when absolutely necessary, as they can clutter code and make it harder to read. Instead, it is important to prioritize clear and descriptive naming conventions as the primary way to describe things in code. This will make the code easier to understand and maintain in the long run 🤓

Reply:
Thank you. I'll change the class names to be more clear.